requirements.txt

```diff
@@ -0,0 +1,9 @@
+# graph-notebook dependencies.
+graph-notebook==3.0.2
+notebook==5.7.10
+tornado==4.5.3
+rdflib==5.0.0
+gremlinpython==3.4.12
+
+# neptune_python_utils dependencies.
+requests==2.26.0
```

Comment:
Is requests a dependency? Shouldn't it be `backoff`?

Reply:
yeah you're right, it's weird because it worked in any case.

Reply:
Ah okey now It was a leftover, I fixed it: backoff==1.11.1